aQute.libg/test/aQute/lib/tag/TagTest.java

```diff
@@ -0,0 +1,289 @@
+package aQute.lib.tag;
+
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.ListIterator;
+import java.util.Map;
+
+import junit.framework.TestCase;
+
+public class TagTest extends TestCase {
+
+	public void testDtoToTag() {
+		Tag t;
+		ListIterator<Object> l;
+		Tag c;
+		Map<String,Object> map;
+		Map<String,Object> map2;
+
+		// Test Scalar
```

Comment:
Can you break this test method in a  number of test methods? Each // comment seems like a good split point.